Dependabot PRs failing due to outdated `pnpm-lock.yaml`

### Description
Dependabot PRs are failing in CI because Dependabot is only updating the `package.json` files but not correctly regenerating the corresponding `pnpm-lock.yaml` files.

Error Message
```
ERR_PNPM_OUTDATED_LOCKFILE  Cannot install with "frozen-lockfile" because pnpm-lock.yaml is not up to date with <ROOT>/sites/xyflow.com/package.json
```

--- a/.github/dependabot.yml
+++ b/.github/dependabot.yml
@@ -5,6 +5,7 @@ version: 2
 updates:
   - package-ecosystem: 'npm'
     directories:
+      - '/' 
       - '/packages/xy-shared'
       - '/sites/reactflow.dev'
       - '/sites/xyflow.com'
@@ -13,7 +14,9 @@ updates:
     allow:
       - dependency-name: '@xyflow/react'
   - package-ecosystem: 'npm'
-    directory: '/sites/svelteflow.dev'
+    directories: 
+      - '/' 
+      - '/sites/svelteflow.dev'
     schedule:
       interval: 'daily'
     allow:

